src/backend/Csrs.Api/Configuration/ConfigurationManagerExtensions.cs

```diff
@@ -1,16 +1,16 @@
-﻿using System.Diagnostics.CodeAnalysis;
+﻿using Csrs.Api.Configuration;
+using System.Diagnostics.CodeAnalysis;
 
-namespace Csrs.Api.Configuration
+namespace Microsoft.Extensions.DependencyInjection;
```

Comment:
Changed namespace so that in Program.cs, we do not require additional using statements and the `AddXXX` method is just available.